Source/Fuse.Maps/iOS/MapViewDelegate.m

```diff
@@ -214,6 +224,48 @@ -(void) clearMarkers
 			[_mapView removeAnnotation:a];
 		}
 		[_annotations removeAllObjects];
+
+		for(id key in _overlays)
+		{
```

Comment:
Ugh, Yeah. We really need to go through and make this code consistent, 'cause now the code-style here is a mess. But that's unrelated to your work, so thanks for bearing with me!